Fish_Doku/Filter_Types/trigonometrics.md

```diff
@@ -84,7 +84,7 @@
 - Type name: `arctangent`
 - Type id: `24`
 - Input channels: 
-  * `value_in` of type `double`
+  * `value_in` of type `double` (default 0)
```

Comment:
We should also issue a warning here. The same goes for the other primary inputs of the trigonometric filters.